docs/LwM2M_Client/STMicroelectronics/STM32L496G-DISCOBG96.md

```diff
@@ -25,15 +25,17 @@ The board is now flashed: you can go to the [Connecting to the LwM2M Server](#co
 !!! Note
     This step is optional. If you've gone through the [Use an already built binary](#use-an-already-built-binary) step, you can go to [Connecting to the LwM2M Server](#connecting-to-the-lwm2m-server) right away.
 
-#### Part 1: Cloning the Anjay freeRTOS client repository
+#### Cloning the Anjay freeRTOS client repository
 
-Enter the command line interface on your machine and paste the following command:
+Enter the command line interface on your machine and run the following command:
 
    ```
    git clone --recursive https://github.com/AVSystem/Anjay-freertos-client
    ```
+!!! Note
+    Only the simplest version of the Anjay Client is described here. To build binary with FOTA go to [Anjay Client with FOTA](#anjay-client-with-fota)
 
-#### Part 2: Compiling the board
+#### Compiling the board
```

Comment:
Looking at this... Do we really compile the board? We compile the application :D This can be changed in a different commit in this MR in all projects.